src/content/docs/usage/runners.mdx

```diff
@@ -0,0 +1,191 @@
+---
+title: Runners
+description: Learn how to use and manage Runners in Daytona.
+---
+
+A Runner is a standalone service that can be deployed anywhere. The user can deploy a Runner on a target system by executing a set of instructions and commands generated upon Runner registration.
+
+Once deployed, the Runner starts up and connects to the [Daytona Server](/docs/usage/server), at which point its status is marked as `Running`.
+
+The Runner sends all requests, while the Server only responds. The Runner sends requests such as _Heartbeat_ (to share its status) and _Job Requests_ (to ask Daytona Server for jobs assigned to it).
+
+A single Runner can use multiple [Providers](/docs/configuration/providers), configured during deployment or later via jobs, enabling flexibility for various environments.
+
+## Register a Runner
+
+Daytona provides an option to register a new Runner, allowing you to deploy it on a target system and connect it to the Daytona Server.
+
+1. Run the following command to register a new Runner:
+
+```shell
+daytona server runner register
+```
+
+Upon running the command, Daytona will generate a code snippet containing the server API URL and API key required to deploy the Runner on the target system.
+
+2. Paste the code snippet into the target system's terminal and execute it to configure the Runner.
+
+## Start a Runner
+
+Daytona provides an option to start a Runner, enabling you to deploy it on the target system and connect it to the Daytona Server.
```

Comment:
I suggest we have `serve` and `start` in this section and make it clear what does what

Maybe we can do it in tabs (`Foreground` | `Daemon/Background`) or just one below the other